internal/websocket/ws.go

```diff
@@ -289,22 +291,32 @@ func (ws *Websocket) run(ctx context.Context) {
 			// Read loop
 			for {
 				var msg wrp.Message
-				ctx, cancel := context.WithTimeout(ctx, ws.inactivityTimeout)
-				typ, reader, err := ws.conn.Reader(ctx)
```

Comment:
we want to include the activity from pings. that requires us to manually trigger a timeout.

Otherwise, device will reconnect in intervals of `ws.inactivityTimeout`